src/plot_dromel_two_pop.R

```diff
@@ -8,7 +8,7 @@ library(RColorBrewer)
 
 
 ## read in data
-setwd("~/Desktop/PopSim/figures/data/")
+setwd("~/Desktop/PopSim/manuscript/data/")
```

Comment:
You should change this to
```
 setwd("../data/")
```
since relative paths are reproducible and absolute ones are not. (that's the path relative to the directory this is in)